app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -241,6 +242,110 @@ public GetMoodTask(String filterBy){
         }
     }
 
+    public static class AddUserTask extends AsyncTask<User, Void, Void> { //TODO: What if username already exists within the database? We need usernames to be UNIQUE for proper sorting.
+
+        @Override
+        protected Void doInBackground(User ... users ) {
```

Comment:
doInBackground is a method that MUST be called since AsyncTask demands it. There is no way that we can "change for a more descriptive name", it is a required method that runs when we call .execute() on an AsyncTask.